### PR TITLE
graalvm8: 19.1.0 -> 19.1.1

### PR DESCRIPTION
**Pull request by volth**
###### Motivation for this change

new version

### CODE REVIEW

File: pkgs/development/compilers/graalvm/default.nix [hunk 1]
```diff
@@ -172,13 +172,13 @@ let
 in rec {
 
   mx = stdenv.mkDerivation rec {
-    version = "5.224.10";
+    version = "5.223.0";
```
Comment (volth) [inline pkgs/development/compilers/graalvm/default.nix]: this is actually a downgrade to use the exact version which 19.1.1 uses
https://github.com/oracle/graal/compare/vm-19.1.0...release/graal-vm/19.1

File: pkgs/development/compilers/graalvm/default.nix [hunk 2]
```diff
@@ -217,13 +217,13 @@ in rec {
   };
 
   jvmci8 = stdenv.mkDerivation rec {
-    version = "20-b04";
+    version = "19.2-b01";
```
Comment (volth) [inline pkgs/development/compilers/graalvm/default.nix]: This is NOT a downgrade, just weird numbeging
19.2-b01 is newer than 20-b04

File: pkgs/development/compilers/graalvm/default.nix [hunk 3]
```diff
@@ -410,7 +410,7 @@ in rec {
       description = "High-Performance Polyglot VM";
       license = licenses.gpl2;
       maintainers = with maintainers; [ volth hlolli ];
-      platforms = [ "x86_64-linux" "aarch64-linux" "x86_64-darwin" ];
+      platforms = [ "x86_64-linux" /*"aarch64-linux" "x86_64-darwin"*/ ];
```
Comment (volth) [inline pkgs/development/compilers/graalvm/default.nix]: Try to make ofborg happy

**Comment by hlolli** [comment]
Sorry for the delay, I'm on a holiday :)

**Pull request merged**